examples/echobot2.py

```diff
@@ -17,9 +17,10 @@
 bot.
 """
 
-from telegram.ext import Updater, CommandHandler, MessageHandler, Filters
 import logging
 
+from telegram.ext import Updater, CommandHandler, MessageHandler, Filters
+
```

Comment:
Please undo this.
It's good, but I prefer to focus on the main subject of the PR. So you can send another PR of style cleanups.
(I'd be more receptive to this if it was a file that you edit anyhow)